app/components/UI/CollectibleContracts/index.test.tsx

```diff
@@ -495,4 +495,124 @@ describe('CollectibleContracts', () => {
     expect(spyOnUpdateNftMetadata).toHaveBeenCalledTimes(0);
     expect(spyOnDetectNfts).toHaveBeenCalledTimes(1);
   });
+
+  it('Should show spinner if nfts are still being fetched', async () => {
```

Comment:
Follow contribution guidelines for test naming please.
```suggestion
  it('Should show spinner if nfts are still being fetched', async () => {
```